index.html

```diff
@@ -352,17 +352,64 @@ <h4>Script Type</h4>
             <p class="issue" data-number="33"></p>
             <p>The definitions of the types of documents and the corresponding <code>ttm:role</code> values are:
               <ul>
-                <li><dfn>Original Language Dialogue List</dfn>: When the <code>ttm:role</code> value is <code>x-DUBBING_ORIGINAL_DIALOGUE_LIST</code>, the document represents a literal transcription of the dialogue and on-screen text in their original spoken/written language(s).
-                <aside class="example">If a movie contains dialogue in English and Hebrew, the <a>Original Language Dialogue List</a> will contain some <a>Script Events</a> in English and some in Hebrew. None of the <a>Script Events</a> is translated.</aside>
+                <li>
+                  <dfn>Original Language Dialogue List</dfn>:
+                  When the <code>ttm:role</code> value is <code>x-DUBBING_ORIGINAL_DIALOGUE_LIST</code>,
+                  the document represents a literal transcription of the dialogue and on-screen text in their original spoken/written language(s).
+                <aside class="note"><a>Text</a> objects in the Original Language Dialogue List have their <a>Text Language Source</a> set to <a>Original</a>.</aside>
```

Comment:
I understand we don't want to have a normative statement in the note, but this makes the sentence ambiguous. Could we have a forward reference to where the actual normative statement is? Something like, "As specified in XXX, Text objects have their TLS set to Original"?

Reply:
We don't have any such normative statements so far. I'm not sure what we could state, normatively?

Reply:
Made an attempt to do this in 3cfe8db - please review. I could only come up with SHOULD / SHOULD NOT and MAY statements, but they're still probably clearer than the notes.